modules/appeals_api/app/models/concerns/appeals_api/appeal_model.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module AppealModel
+    extend ActiveSupport::Concern
+
+    included do
+      scope :without_status_updates_since, lambda { |time|
+        status_update_table = AppealsApi::StatusUpdate.table_name
+        join_clause = <<~JOIN
+          LEFT JOIN #{status_update_table}
+          ON #{table_name}.id = CAST(#{status_update_table}.statusable_id as uuid)
+          AND #{status_update_table}.id IS NULL
+          AND #{status_update_table}.statusable_type = '#{sanitize_sql(klass.name)}'
+        JOIN
+        where("#{table_name}.updated_at <= ?", time)
```

Comment:
Note that here, when fetching appeals `without_status_updates_since(...)`, I'm also rejecting appeals that have any updates to the model since the datetime in question - so if the appeal has had any status updates **OR updates to the model** after the datetime supplied, it will not be included in results. This wasn't explicitly specified in the requirements in the ticket, but it seemed to follow from them. For example, this would prevent an appeal that has _no_ status updates at all from being cleared of PII before the specified datetime.